app/grandchallenge/reader_studies/forms.py

```diff
@@ -142,6 +143,30 @@ class Meta(ReaderStudyCreateForm.Meta):
         }
 
 
+class ReaderStudyCopyForm(Form):
+    copy_images = BooleanField(required=False, initial=True)
+    copy_hanging_list = BooleanField(required=False, initial=True)
+    copy_case_text = BooleanField(required=False, initial=True)
+    copy_questions = BooleanField(required=False, initial=True)
+    copy_readers = BooleanField(required=False, initial=True)
+    copy_editors = BooleanField(required=False, initial=True)
```

Comment:
Can we set the title of the new reader study here too as this sets the slug, which is not editable by the user.